addons/medical_engine/functions/fnc_handleDamage.sqf

```diff
@@ -67,6 +67,34 @@ if (
     0
 };
 
+// Being inside an exploding vehicle doesn't trigger for each hitpoint
+// It seems to fire twice with ammo type "FuelExplosion" or "FuelExplosionBig"
+if (
+    _hitPoint isEqualTo "#structural" &&
+    {_ammo isKindOf "FuelExplosion"} &&
+    {_vehicle != _unit} &&
+    {_damage == 1}
```

Comment:
Is it always 1?

Reply:
Yes, at least in all the testing I've done.
Also, not having this condition could lead to double-handling when a unit is inside a vehicle and another vehicle explodes nearby.